package/yast2.changes

```diff
@@ -1,4 +1,16 @@
 -------------------------------------------------------------------
+Tue Jun  6 10:33:22 UTC 2017 - jreidinger@suse.com
+
+- Fix showing help text when CWM::ReplacePoint contain another
+  CWM::ReplacePoint. Fix include new
+  CWM::AbstractWidget#dynamic_help functionality for widgets where
+  help text can change during its lifetime. Fix also include new
+  variant of Yast::CWM.ReplaceWidgetHelp without arguments, that
+  recompute help, which should be called if help text changed in
+  widgets with CWM::AbstractWidget#dynamic_help. (boo#1039901)
```

Comment:
(This will need an update. Remember http://tinyurl.com/this-verb-form-are-wrong ;-) )